addons/vrm/vrm1_collider.gd

```diff
@@ -0,0 +1,89 @@
+@tool
+extends Resource
+
+# Bone name references are only valid within the given Skeleton.
+# If the node was not a skeleton, bone is "" and contains a path to the node.
+@export var skeleton_or_node: NodePath
+
+# The bone within the skeleton with the collider, or "" if not a bone.
+@export var bone: String
+
+@export var offset: Vector3
+@export var tail: Vector3 # if is_capsule
+@export var radius: float
+
+@export var is_capsule: bool = false
+
+# (Array, Plane)
+# Only use in editor
+@export var gizmo_color: Color = Color.MAGENTA
+
+#func setup():
+#	if parent != null:
+#		colliders.clear()
+#		for collider in sphere_colliders:
+#			colliders.append(SphereCollider.new(bone_idx, collider.normal, collider.d))
+
+
+#func _ready(ready_parent: Node3D, ready_skel: Object):
+#	self.parent = ready_parent
+#	if ready_parent.get_class() == "Skeleton3D":
+#		self.skel = ready_skel
+#		bone_idx = ready_parent.find_bone(bone)
+#	setup()
+
+
+#func _process():
+#	for collider in colliders:
+#		collider.update(parent, skel)
+
```

Comment:
Unused code.

Reply:
spring bones are not yet implemented. this is to-be-written not unused.